### PR TITLE
Updates for Tensorflow 1.0

### PR DESCRIPTION
**Pull request by henryre**
* Updates loss function to TF 1.0 (LR and reLSTM)
* Removes regularization from bias term (LR)
* Adds thread bounding (TFNoiseAwareModel)
* Switches to Adam optimizer (LR)
* Adds explicit seeding for parameter initialization (LR)
* Fixes RNN library (reLSTM)

### CODE REVIEW

**Comment by ajratner** [comment]
@henryre ready for review?

**Comment by henryre** [comment]
Nah some versioning issue

**Comment by ajratner** [comment]
ok ping me when ready!

**Comment by henryre** [comment]
OK good to go! Should fix everything for TF 1.0
